noetic/distribution.yaml

```diff
@@ -173,6 +173,31 @@ repositories:
       url: https://github.com/ros-controls/control_msgs.git
       version: kinetic-devel
     status: maintained
+  diagnostics:
+    doc:
+      type: git
+      url: https://github.com/ros/diagnostics.git
+      version: noetic-devel
+    release:
+      packages:
+      - diagnostic_aggregator
+      - diagnostic_analysis
+      - diagnostic_common_diagnostics
+      - diagnostic_updater
+      - diagnostics
+      - rosdiagnostic
+      - self_test
+      - test_diagnostic_aggregator
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/ros-gbp/diagnostics-release.git
+      version: 1.9.4-0
```

Comment:
Ooops, the source entry below is all that's needed to enable PR testing The doc block won't hurt anything, but the release block should be deleted.